### PR TITLE
ENH: Clarify shape of PDT2 images and recommend acq entity for split PDw/T2w images

### LINKED ISSUE #223: PDT2 anatomical modality: Recommendations and potential use of _echo
For PD+T2 images created from a single series (two echos), our conversion tool (heudiconv) is encouraging us to use `_PDT2` for modality type, creating two NIFTI+json pairs, one for PD and one for T2, but we're struggling to figure out how to give them distinct names.

I don't recall a discussion around the `PDT2` anatomy modality label: What was intended?  That a 4D (dim4=2) image would be created?  

If two 3D images are created, how should they be distinguished?  According to the spec, the only feasible options are `_acq-<label>` and `_run-<index>`.  `run` doesn't really make sense as it suggests that two `PDT2` acquisitions were collected.  So might `_acq-PD` and `_acq_T2w` be recommended?  

What would be unambiguous and perhaps clearer is if `_echo-<index>` was available as an option.  Is there any interest/appetite for adding an optional `echo` keyword for anatomical scans?

Anyway, the PD+T2 is a very common clinical acquisition and if nothing else it would be good to offer some concrete suggestions in the spec.

### PR DESCRIPTION
**Pull request by effigies**
To briefly summarize the discussion in #223, PDT2 sequences are common in clinical settings, taking advantage of view-sharing processes to simultaneously acquire PDw and T2w volumes. It is currently ambiguous how these files should be named if the volumes are split into individual NIfTI files, which is the default behavior of BIDS converters such as dcm2niix.

It was initially proposed to add the `echo-` entity, which would be correct but unique among anatomical images. It was objected that clinical users may not easily recognize whether `echo-1_PDT2` indicates the PDw or T2w image. On the other hand, it is possible to have PDw or T2w images in addition to PDT2 images, and recommendations for consistent unambiguous naming would be desirable.

This change intends to *clarify* that PDT2 images SHOULD be 4D with length 2. If PDT2 images are split, their suffixes SHOULD be PDw and T2w, respectively, and we suggest using the `acq-` parameter to indicate they come from the same source. @tiborauer suggested `acq-partial`; here I propose `acq-PDT2` to be very explicit.

----

The expected impact of this is low, as PDT2 files are rare outside clinical contexts. None appear in OpenNeuro unless they have already been renamed according to the proposed convention.

The practical consequence for the validator are:

1) Add a warning if PDT2 images do not have 2 volumes. This is done in schema, and I do not currently intend to backport it to the legacy validator.
2) Remove support for PDT2map, a modality that is not believed to actually exist anywhere. This may need to be reverted and PDT2map re-added to the standard if it turns out that anybody is using it.

Closes #223.

### CODE REVIEW

**Comment by tiborauer** [comment]
Let me add a short elaboration. I suggested we do not have to treat these images any differently than other T2w and PDw images and use the `acq` field, as usual, to differentiate them.  I suggested `acq-partial` only as an example and only if it their most relevant and distinctive feature (i.e., partial brain coverage), as suggested by @CPernet.

**Comment by effigies** [comment]
Thanks Tibor. I agree. If there is a wording improvement to make that clearer, please feel free to suggest it.

File: src/schema/objects/suffixes.yaml [hunk 1]
```diff
@@ -207,17 +207,12 @@ PDT2:
...
-  description: |
-    In arbitrary units (arbitrary).
-    Combined PD/T2 maps are REQUIRED to use this suffix regardless of the method
-    used to generate them.
+    A two-volume 4D image, where the volumes are, respectively, PDw and T2w
+    images acquired using a dual-echo FSE sequence through a view sharing process
+    (for example, [Johnson et al. 1994](https://pubmed.ncbi.nlm.nih.gov/8010268/)).
+    If separated into 3D volumes, the `PDw` and `T2w` suffixes SHOULD be used
+    instead, and an acquisition entity MAY be used to indicate the origin of the
+    images, for example, `acq-PDT2_PDw.nii` and `acq-PDT2_T2w.nii`.
```
[the remark this answers is not in the document]
Comment (tiborauer) [reply src/schema/objects/suffixes.yaml:215]: Images with the same suffixes MUST rather than MAY be distinguished.

**Review by nicholst — APPROVED** [review]
Thanks @effigies for creating this PR, and for updating it as per my comments in #223.  I've reviewed it carefully and it looks good to me.

**Review by tiborauer — APPROVED** [review]

**Review by CPernet — APPROVED** [review]

**Review by agahkarakuzu — APPROVED** [review]

**Comment by effigies** [comment]
Thanks for the reviews, all!

**Pull request merged**